corpus/statements.txt

```diff
@@ -460,6 +460,7 @@ Variable declarations
 
 var x = 1;
 var x, y = {}, z;
+var set = 1;
```

Comment:
I think this test would pass on master, because the `set` token is not valid in this context. Is there some case that you're seeing in the wild that does not parse successfully on master? It would be good to tweak this test to cover that case.

Reply:
It's ok with master so I just keep the test.